src/cobalt_model/pagination_config.rs

```diff
@@ -0,0 +1,59 @@
+use std::convert::Into;
+use std::vec::Vec;
+
+use super::SortOrder;
+
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub enum Include {
+  None,
+  #[serde(rename = "all")]
```

Comment:
Rust styling force capital for the first char. But it feels odd to me to write `include: All` instead of `include: all`. Maybe it's just me though, let me know what you prefer :)